### PR TITLE
Fix when using the flag --custom_builds_dir-enabled

### PR DESCRIPTION
**Pull request by krousenick**
This option for the gitlab-runner register command does not take any options/arguments. If you set it to true it breaks the registration of the runner.

### CODE REVIEW

File: .ansible-lint [hunk 1]
```diff
@@ -3,3 +3,4 @@ profile: production
 warn_list:
   - role-name  # until role name is fixed on galaxy https://github.com/riemers/ansible-gitlab-runner/pull/312
   - no-changed-when  # TODO in future
+  - yaml[line-length]
```
Comment (guenhter) [inline .ansible-lint:6]: I'm not a fan of this. Instead we should try to fix the actual lint violations in a clean manner.

File: tasks/register-runner.yml [hunk 2]
```diff
@@ -138,7 +138,7 @@
       --builds-dir '{{ gitlab_runner.builds_dir }}'
       {% endif %}
       {% if gitlab_runner.custom_build_dir_enabled | default(false) %}
-      --custom_build_dir-enabled '{{ gitlab_runner.custom_build_dir_enabled }}'
+      --custom_build_dir-enabled
```
Comment (guenhter) [inline tasks/register-runner.yml:141]: Is this working without a true|false argument?

Comment (krousenick) [reply tasks/register-runner.yml:141]: Yes. There is no true/false option with the switch.

Comment (krousenick) [reply tasks/register-runner.yml:141]: I verified with 'gitlab-runner register --help' lol

**Comment by guenhter** [comment]
LGTM.

When you are ready, I'm happy to merge.

**Comment by krousenick** [comment]
This should also fix the ansible lint error for the 160 characters thing in defaults/main.yml:16. Take a look and let me know if you're ok with that.

**Comment by guenhter** [comment]
Very nice. Really appreciate that.

**Comment by krousenick** [comment]
Ready to merge if you are!

**Comment by guenhter** [comment]
Thx for this contribution

**Pull request merged**